### PR TITLE
Bolsalice keyboard creation

### PR DESCRIPTION
**Pull request by stanrc85**
Adding Bolsalice keyboard.

PCB and source code by @CMMS-Freather, with some minor updates by me.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/bolsa/bolsalice/config.h [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4253
+#define PRODUCT_ID      0x414C
+#define DEVICE_VER      0x0001
```
Comment (drashna) [inline keyboards/bolsa/bolsalice/config.h:24]: Please move this to the info.json file.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/bolsa/bolsalice/config.h (hunk 1 above)
Comment (stanrc85) [reply keyboards/bolsa/bolsalice/config.h]: Done!

**Review by drashna — APPROVED** [review]

File: keyboards/bolsa/bolsalice/info.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    "manufacturer": "BolsaSupply",
+    "usb": {
+        "pid": "0x414C",
+        "vid": "0x4253",
+        "device_version": "0.0.1"
+    },
+    "url": "",
+    "maintainer": "qmk",
+    "layouts": {
+        "LAYOUT": {
```
Comment (fauxpark) [inline keyboards/bolsa/bolsalice/info.json:12]: The layouts you have in here and the keyboard.h do not make sense.
image

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/bolsa/bolsalice/info.json (hunk 2 above)
Comment (stanrc85) [reply keyboards/bolsa/bolsalice/info.json]: They've been fixed now

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**